rtsp.go

```diff
@@ -21,19 +21,19 @@ import (
 	goutils "go.viam.com/utils"
 
 	"go.viam.com/rdk/components/camera"
-	"go.viam.com/rdk/components/camera/rtsp"
```

Comment:
This package will eventually go away. Config used to be : https://github.com/viamrobotics/rdk/blob/0a7f6b23851f8c04f82ffa5583f600668e5e2e3a/components/camera/rtsp/rtsp.go#L46